modules/rational-ui.el

```diff
@@ -10,6 +10,20 @@
 ;; User interface customizations. Examples are the modeline and how
 ;; help buffers are displayed.
 
+;; This package provides a basic, customized appearance for
+;; Emacs. Specifically, it uses: Helpful to customize the information
+;; and visual display of help buffers, such as that created by M-x
+;; `describe-function'; Doom Modeline and Themes, to customize the
+;; appearance of buffers, text, et cetera; All-the-icons, to provide
+;; Doom Modeline with font-based icons (rather than raster or vector
+;; images); and includes some Emacs Lisp demonstrations.
+
+;;  Run `all-the-icons-install-fonts' to ensure the fonts necessary
+;; for ALL THE ICONS are available on your system. You must run this
+;; function if the "stop" icon at the beginning of this paragraph is
+;; not displayed properly (it appears as a box with some numbers
+;; and/or letters inside it).
+
```

Comment:
I'd probably add that you need to install the *downloaded* fonts on some platforms manually (e.g. Windows). That has been an issue in the past, see #80.

Reply:
Can the `ON-WINDOWS` constant be used to signal that?

Reply:
Sure. Something along

```lisp
(when ON-WINDOWS
  (advice-add ....))
```
However, I wouldn't bother with that in this PR; the current PR is fine as-is, that's more or less an idea if you (or anyone else) wants to further improve ergonomics :)

Reply:
Resolved by #97.